File: Bronze4/js/백준 17388번/B_17388.js
const fs = require('fs');
let ip = fs.readFileSync('B_17388.txt').toString().split(' ');
var a = parseInt(ip[0]);
var b = parseInt(ip[1]);
var c = parseInt(ip[2]);
var sum = a + b + c;
if (sum >= 100) console.log("OK");
else {
    var min = Math.min(a, Math.min(b, c));
    if (min == a) console.log("Soongsil");
    else if (min == b) console.log("Korea");
    else console.log("Hanyang");
}

File: Bronze4/Python/B_17388.py
s, k, h = map(int, input().split())
if s + h + k >= 100:
    print('OK')
else:
    if min(s, k, h) == s:
        print('Soongsil')
    elif min(s, k, h) == k:
        print('Korea')
    else:
        print('Hanyang')